### DISCUSSION TITLE
Modal doesn't work

### DISCUSSION THREAD

**Discussion by CzachurskiVeronica** [alpinejs/alpine#4085] [1. Help]
I'm working in Livewire 3.x, I'm trying to insert Alpine js for the events.
This is the version of Alpinejs in the folder app.blade.js

        
    <script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/focus@3.x.x/dist/cdn.min.js"></script>
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
    
In the view for the route tipos/tipos.blade.php i call the component to create tipos

<div>
    <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8 py-12">

    <body class="antialiased font-sans bg-gray-200">
            <div class="container mx-auto px-4 sm:px-8">
                <div class="py-8">
                    <div>
                        <h2 class="text-2xl font-semibold leading-tight">TIPOS</h2>
                    </div>
                    <div class="px-6 py-4 flex items-center">
                        <x-input class="flex-1 mr-4" placeholder="Búsqueda por sigla/nombre" type="text" wire:model.live="search" />
                        @livewire('tipos.create-tipos')
                    </div>
          
          
      
![image](https://github.com/alpinejs/alpine/assets/137120296/9e0e9b37-db89-429a-b614-e2bdfa3bb631)

When the page loads the modal is always open. But when I click on the button the modal doesn't open. I have to make changes but i don't know which ones

**Comment by joshhanley** [comment]
@CzachurskiVeronica Livewire v3 already comes bundled with Alpine so you need to remove your Alpine script tags as it will be conflicting.

**Comment by ekwoka** [comment]
your x-show expression is incorrect

`open = true` sets open to true. It does not check if open is true.
